sdk/formrecognizer/Azure.AI.FormRecognizer/src/Generated/Models/TextLine.cs

```diff
@@ -46,7 +45,7 @@ internal TextLine(string text, IEnumerable<float> boundingBox, IEnumerable<TextW
         /// <param name="language"> The detected language of this line, if different from the overall page language. </param>
         /// <param name="words"> List of words in the text line. </param>
         /// <param name="appearance"> Text appearance properties. </param>
-        internal TextLine(string text, IReadOnlyList<float> boundingBox, Language? language, IReadOnlyList<TextWord> words, Appearance appearance)
+        internal TextLine(string text, IReadOnlyList<float> boundingBox, Language? language, IReadOnlyList<TextWord> words, TextAppearance appearance)
```

Comment:
Do we do the `Appearance` to `TextApperarance`  changes in the `Generated/Models` itself for .NET?

Reply:
yes. Because the content itself doesn't change, I just rename the generated model. :)